wild/tests/integration_tests.rs

```diff
@@ -2425,6 +2426,35 @@ fn integration_test(
     Ok(())
 }
 
+#[rstest]
+#[cfg(feature = "mold_tests")]
+fn exec_mold_tests(
+    #[files("../external_test_suites/mold/test/*.sh")] mold_test: PathBuf,
+) -> Result {
+    let path = env::var("PATH").unwrap();
```

Comment:
I know it's very unlikely the `env` is not set, but I would rather add `.context` and use `?` (instead of the unwrap);